packages/camera/camera_android_camerax/CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to camera\_android\_camerax
+
+## Plugin structure
```

Comment:
not a blocker but I think this should start with an overview of the architecture @bparrishMines designed. Maybe linking the design doc or if not that then a 1 paragraph summary describing the structure.

Reply:
Added one!